File: app/src/androidTest/java/net/tlalka/android/fiszki/view/pages/LessonListPage.java
package net.tlalka.android.fiszki.view.pages;

import net.tlalka.android.fiszki.R;

public class LessonListPage extends AbstractPage {

    @Override
    public void valid() {
        isVisible(R.id.lesson_icon);
        isVisible(R.id.lesson_topic);
        isVisible(R.id.lesson_progress);
        isVisible(R.id.lesson_list_view);
    }

    public void openItem(int index) {
        clickListItem(R.id.lesson_list_view, index);
    }
}

File: app/src/androidTest/kotlin/net/tlalka/fiszki/view/pages/LessonListPage.kt
package net.tlalka.fiszki.view.pages

import net.tlalka.fiszki.R

class LessonListPage : AbstractPage() {

    override fun valid() {
        isVisible(R.id.lesson_icon)
        isVisible(R.id.lesson_topic)
        isVisible(R.id.lesson_progress)
        isVisible(R.id.lesson_list_view)
    }

    fun openItem(index: Int) {
        clickListItem(R.id.lesson_list_view, index)
    }
}
